### ISSUE TITLE
Refactor CSS to use common variables

### ISSUE THREAD

**Issue by jimporter** [mozilla-lockbox/lockbox-extension#319]
It'd be nice if we used CSS variables for our colors. This will also probably require that we add CSS minification to remove duplicate definitions of said variables.

**Comment by devinreams** [comment]
>It'd be nice if we used CSS variables for our colors

@jimporter Agreed. What do you think about using the variables approach documented here? https://github.com/css-modules/css-modules/blob/master/docs/values-variables.md

**Comment by pdehaan** [comment]
How about using Sass? That way we'd get improved nesting support and other stuff for "free".

**Comment by jimporter** [comment]
I don't think we get much out of SASS that we don't already get from modularized CSS.
